docs/guides/data-loading.md

```diff
@@ -169,8 +169,8 @@ If you picked Cloudflare Workers as you environment, [Cloudflare Key Value][clou
 import { useLoaderData } from "remix";
 import type { LoaderFunction } from "remix";
 
-export let loader: LoaderFunction = async ({ params }) => {
-  return PRODUCTS_KV.get(`product-${params.productId}`, {
+export let loader: LoaderFunction = async ({ env, params }) => {
+  return env.PRODUCTS_KV.get(`product-${params.productId}`, {
```

Comment:
Isn't the `env` property within the `context` object?

```suggestion
export let loader: LoaderFunction = async ({ context, params }) => {
  return context.env.PRODUCTS_KV.get(`product-${params.productId}`, {
```

Reply:
I don't believe so. The `context` is the parameter of the loader function and `env`, `params`, etc are being destructured out of it.

`export let loader: LoaderFunction = async ({ env, params }) => {` 

is equivalent to

```
export let loader: LoaderFunction = async (context) => {
const { env, params } = context;
}
```

Reply:
But the `LoaderFunction` type has `DataFunctionArgs` as the type for the arguments:

```ts
export interface DataFunctionArgs {
    request: Request;
    context: AppLoadContext;
    params: Params;
}
```

Therefore: 
![image](https://user-images.githubusercontent.com/770632/151681679-dfc24a8a-b950-45e0-ba23-30db3a6875eb.png)

Reply:
yep, this needs to be on `context.env`

Reply:
Great job catching my mistake Ponjimon and thanks for the clarification Ryan.

Thanks!